IDE/Espressif/ESP-IDF/examples/wolfssl_benchmark/main/helper.c

```diff
@@ -78,3 +163,30 @@ int construct_argv()
 
     return (cnt);
 }
+
+/* entry point */
+void app_main(void)
+{
+    ESP_LOGI(TAG, "Start app_main...");
+    ESP_ERROR_CHECK(nvs_flash_init());
+    
+
+    ESP_LOGI(TAG, "Start benchmark..");
+    wolf_benchmark_task( );
```

Comment:
Is it possible to cleanup this and remove the `wolf_benchmark_task` call here and in the benchmark.c and just use `benchmark_test(NULL);` below? Also these calls should be wrapped in `#ifndef NO_CRYPT_BENCHMARK`

Reply:
I'd like to call wolf_benchmark_task() rather than benchmark_test(NULL) here because I'd like to pass arguments. This is nicer for me when I want to only see a part of benchmark.

Removed #ifndef [WOLFSSL_ESPIDF](https://github.com/miyazakh/wolfssl/blob/7db485d8b505f0ecb3359b5198c4957e8830f189/wolfcrypt/benchmark/benchmark.c#L5494) in wolf_benchmark_task().
Surrounded NO_CRYPT_BENCHARK.